<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Wintermute Developers &lt;dev@thesii.org&gt;</author>
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Wintermute - Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/img/logo</normaloff>:/img/logo</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>448</y>
     <width>441</width>
     <height>26</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>441</width>
     <height>441</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabOverview">
    <attribute name="title">
     <string>Overview</string>
    </attribute>
    <widget class="QPushButton" name="btnRefresh">
     <property name="geometry">
      <rect>
       <x>348</x>
       <y>376</y>
       <width>81</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Refresh</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="lblOverview">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>417</width>
       <height>385</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%p symbols
%l links
%c concepts
%o ontologies
%n total nodes</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>lblOverview</zorder>
    <zorder>btnRefresh</zorder>
   </widget>
   <widget class="QWidget" name="tabLing">
    <attribute name="title">
     <string>Linguistics</string>
    </attribute>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>429</width>
       <height>397</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabLingMod">
      <attribute name="title">
       <string>Creation + Modification</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>5</y>
         <width>421</width>
         <height>353</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayoutMod">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::ExpandingFieldsGrow</enum>
        </property>
        <property name="horizontalSpacing">
         <number>6</number>
        </property>
        <property name="verticalSpacing">
         <number>6</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="localeLabel">
          <property name="text">
           <string>Locale</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="localeComboBox">
          <property name="frame">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="symbolLabel">
          <property name="text">
           <string>Symbol</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="symbolLineEdit">
          <property name="minimumSize">
           <size>
            <width>273</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>273</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="linksLabel">
          <property name="text">
           <string>Links</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTableWidget" name="tableWidgetLinks">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButtonAdd">
            <property name="text">
             <string>Append</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonUpdate">
            <property name="text">
             <string>Update</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLingDel">
      <attribute name="title">
       <string>Deletion</string>
      </attribute>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>236</x>
         <y>328</y>
         <width>186</width>
         <height>29</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <property name="horizontalSpacing">
         <number>14</number>
        </property>
        <property name="verticalSpacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QPushButton" name="pushButtonDelete">
          <property name="text">
           <string>Delete...</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonSelect">
          <property name="text">
           <string>Select...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>4</x>
         <y>328</y>
         <width>164</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="labelComboBoxLingDelete">
          <property name="text">
           <string>Locale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxLingDeleteLocale">
          <property name="minimumSize">
           <size>
            <width>111</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>111</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTreeWidget" name="treeWidgetDelete">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>421</width>
         <height>321</height>
        </rect>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerHighlightSections">
        <bool>true</bool>
       </attribute>
       <attribute name="headerShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </widget>
     <widget class="QWidget" name="tabLingSerial">
      <attribute name="title">
       <string>Saving + Loading</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>5</y>
         <width>417</width>
         <height>293</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::FieldsStayAtSizeHint</enum>
        </property>
        <property name="formAlignment">
         <set>Qt::AlignCenter</set>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelSaveURI">
          <property name="text">
           <string>URI</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLineEdit" name="uriLineEdit">
            <property name="minimumSize">
             <size>
              <width>118</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>118</width>
              <height>23</height>
             </size>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLingSaveURI">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>23</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelLocale">
          <property name="text">
           <string>Locale</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QComboBox" name="cmbBxLingSaveLocales">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>151</width>
            <height>23</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>151</width>
            <height>23</height>
           </size>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="labelMethod">
          <property name="text">
           <string>Method</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QRadioButton" name="radioButtonSaving">
          <property name="text">
           <string>Saving</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QRadioButton" name="radioButtonLoading">
          <property name="text">
           <string>Loading</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="btnLingSerialDo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>&amp;Execute</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tabSemantics">
    <attribute name="title">
     <string>Semantics</string>
    </attribute>
    <widget class="QTreeView" name="treeViewConcepts">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>429</width>
       <height>401</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabPlugins">
    <attribute name="title">
     <string>Plugins</string>
    </attribute>
    <widget class="QListWidget" name="listWidgetPlugins">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>296</y>
       <width>429</width>
       <height>105</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="formLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>100</y>
       <width>429</width>
       <height>193</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Version:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Vendor:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Description:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Web Page:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QTextEdit" name="txtDescription"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lblVendor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lblVersion">
        <property name="text">
         <string/>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lblWebPage">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Author:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lblAuthor">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lblName">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>8</y>
       <width>425</width>
       <height>57</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>4</x>
       <y>72</y>
       <width>429</width>
       <height>25</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="checkBoxAutoStart">
        <property name="text">
         <string>Auto Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBoxEnabled">
        <property name="text">
         <string>Enabled</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
